tests/organizations/test_api_get_organization.py

```diff
@@ -0,0 +1,158 @@
+import unittest
+from http import HTTPStatus, cookies
+from unittest.mock import patch, Mock
+import requests
+from requests.exceptions import HTTPError
+from flask import json
+from flask_restx import marshal
+from app import messages
+from app.database.sqlalchemy_extension import db
+from tests.base_test_case import BaseTestCase
+from app.api.request_api_utils import post_request, get_request, BASE_MS_API_URL, AUTH_COOKIE
+from app.api.models.user import full_user_api_model
+from app.api.models.organization import get_organization_response_model, update_organization_request_model
+from tests.test_data import user1
+from app.database.models.ms_schema.user import UserModel
+from app.database.models.bit_schema.user_extension import UserExtensionModel
+from app.database.models.bit_schema.organization import OrganizationModel
+
+
+class TestGetOrganizationApi(BaseTestCase):
+    
+    def setUp(self):
+        super(TestGetOrganizationApi, self).setUp()
+        self.test_user = UserModel(
+            name=user1["name"],
+            username=user1["username"],
+            password=user1["password"], 
+            email=user1["email"], 
+            terms_and_conditions_checked=user1["terms_and_conditions_checked"]
+        )
+        self.test_user.need_mentoring = user1["need_mentoring"]
+        self.test_user.available_to_mentor = user1["available_to_mentor"]
+        self.test_user.save_to_db()
+
+        self.test_user_data = UserModel.find_by_email(self.test_user.email)
+        
+        AUTH_COOKIE["Authorization"] = "this is fake token"
+        AUTH_COOKIE["Authorization"]["expires"] = 1601478236
+        AUTH_COOKIE["user_id"] = self.test_user_data.id
+
+
+    @patch("requests.get")
+    def test_api_dao_get_organization_successfully(self, mock_get_user):
+        # prepare existing organization
+        organization = OrganizationModel(
+            rep_id=int(AUTH_COOKIE["user_id"].value), 
+            name="Company ABC",
+            email="companyabc@mail.com",
+            address="506 Elizabeth St, Melbourne VIC 3000, Australia",
+            website="https://www.ames.net.au",
+            timezone="ALASKA_STANDARD_TIME",
+        )
+        organization.rep_department = "H&R Department"
```

Comment:
don't use rep please. Remove all reps, make them representative or whatever the full form is. Somebody besides you is going to have a real hard time figuring what rep is

Reply:
this one is unavoidable (as per the other remaining rep_.. that is part of database attribute). I have changed the models and other that not related to the database. If you still insist on the rep_ to be changed on the database model, I need to redo not only the Organization and Program models, but also recreate tables on the database. The samee need to be done with the remote database on Heroku. 
So, do we wanna go that far with the remaining of GSoC (restructuring the database)? Or can we put this in icebox to be pick up after GSoC. 
PS: Before answering this, keep in mind I still have lots of deliverables untouched such as the Program, Applying to programs, Mentorship relation, static pages for Homepage (About, Benefit, Contact), Portfolios (User, Organization and Program) and My Spcace Request History.